pom.xml

```diff
@@ -74,6 +77,7 @@
             <version>8.0.12</version>
         </dependency>
 
+        <!--Хрен знает что-->
```

Comment:
If the app works without this dependency, remove it.
If not, try to understand what does it mean.